src/index.js

```diff
@@ -159,7 +179,7 @@ export const useReduxState = () => {
     const unsubscribe = store.subscribe(callback);
     return unsubscribe;
   }, [store]); // eslint-disable-line react-hooks/exhaustive-deps
-  return proxyfied.trappedState;
+  return trapped.state;
```

Comment:
`trapped.reset` just before return to start a new render from a blank list?

Reply:
same here. unless you want to call `reset()` to newly created `trapped`, it should be ok.